src/pages/Embeds/Plio.vue

```diff
@@ -656,12 +700,53 @@ export default {
       // insert the button inside the plyr instance so that it shows up in fullscreen mode
       this.$nextTick(() => {
         let maximizeButton = document.getElementById("plioMaximizeButton");
-        if (maximizeButton != undefined) this.mountOnFullscreenPlyr(maximizeButton);
+        if (maximizeButton != undefined)
+          this.mountOnFullscreenPlyr(maximizeButton);
       });
     },
+    isMovingToTimestampAllowed(timestamp) {
+      if (this.firstUnansweredItem == null) return true;
+      // check whether skip is disabled and all items have been answered
+      return timestamp < this.firstUnansweredItem.time;
+    },
+    /**
+     * whether the user should be moved to the first unanswered item
+     * or allowed to pass through to whatever timestamp they are trying to
+     * go to
+     */
+    moveToFirstUnansweredItemOrPass() {
+      if (this.isSkipEnabled) return false;
+      let timeToInspect = this.player.currentTime;
+
+      /**
+       * this is required to handle the case that the user toggles
+       * the show video/question button while an item is active and tries
+       * to play the video since the user will be in the time range of item
+       * timestamp and item timestamp - POP_UP_CHECKING_FREQUENCY, the code
+       * after this snippet will function incorrectly
+       */
```

Comment:
```suggestion
      /**
       * this is required to handle the case when the user toggles
       * the show video/question button while an item is active and tries
       * to play the video. 
       * since the user will be in the time range of item timestamp and 
       * item timestamp - POP_UP_CHECKING_FREQUENCY, the code
       * after this snippet will function incorrectly
       */
```